src/apps/frontend/types/auth.ts

```diff
@@ -3,10 +3,12 @@ import { JsonObject } from './common-types';
 export class AccessToken {
   accountId: string;
   token: string;
+  expiresAt:string
```

Comment:
We need to pay attention to data modelling. We use Typescript so as to make it easy to reason with it.

`expiresAt` type should not be a string, rather a `Date` object. Server has to return string as it needs to serialize it over the wire.